Lib/dis.py

```diff
@@ -107,13 +112,19 @@ def pretty_flags(flags):
     return ", ".join(names)
 
 def _get_code_object(x):
-    """Helper to handle methods, functions, generators, strings and raw code objects"""
+    """Helper to handle methods, functions, generators,
```

Comment:
Maybe just turn this docstring in a comment? But we should solve the similar issue with the `dis()` docstring.